src/axiom/cognitive_agent.py

```diff
@@ -739,13 +1004,19 @@ def _load_agent_state(self) -> None:
         """
         if os.path.exists(self.state_file):
             try:
-                with open(self.state_file) as f:
+                with open(self.state_file, encoding="utf-8") as f:
```

Comment:
Using encoding is very important, this is a good change. When left unspecified, encoding defaults to platform default encoding. On almost all modern platforms, this is `utf-8`, and there are no issues, but on some platforms the default is a different encoding, and this can lead to errors if you were to try to load a file saved on one computer from a different computer.